e2e-tests/e2e-fixtures/mock-gef-facilities.js

```diff
@@ -226,6 +221,9 @@ const anUnissuedCashFacilityWith20MonthsOfCover = ({ facilityEndDateEnabled = fa
   coverDateConfirmed: false,
   canResubmitIssuedFacilities: null,
 });
+exports.anUnissuedCashFacility = anUnissuedCashFacility;
+exports.anIssuedCashFacility = anIssuedCashFacility;
+exports.anIssuedCashFacilityWithCoverDateConfirmed = anIssuedCashFacilityWithCoverDateConfirmed;
```

Comment:
Can we move these to after `anIssuedContingentFacility` please, concern is these could get lost if new functions/objects are added